.buildkite/scripts/steps/functional/scalability_dataset_extraction.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+USER_FROM_VAULT="$(retry 5 5 vault read -field=username secret/kibana-issues/dev/apm_parser_performance)"
+PASS_FROM_VAULT="$(retry 5 5 vault read -field=password secret/kibana-issues/dev/apm_parser_performance)"
+ES_SERVER_URL="https://kibana-ops-e2e-perf.es.us-central1.gcp.cloud.es.io:9243"
+JOB_ID=${BUILDKITE_JOB_ID}
+
+.buildkite/scripts/bootstrap.sh
+
+journeys=("login" "ecommerce_dashboard" "flight_dashboard" "web_logs_dashboard" "promotion_tracking_dashboard" "many_fields_discover")
+
+for i in "${journeys[@]}"; do
+    JOURNEY_NAME="${i}"
+    echo "Looking for JOURNEY ${JOURNEY_NAME} and JOB_ID ${JOB_ID} in APM traces"
+
+    yarn cli -u "${USER_FROM_VAULT}" -p "${PASS_FROM_VAULT}" -c "${ES_SERVER_URL}" -s "2022-04-12T00:01:00.000Z" -j "${JOB_ID}" -n "${JOURNEY_NAME}"
```

Comment:
Have you added an appropriate script to the `package.json`? Please, be aware that I renamed the bin command https://github.com/elastic/apm-parser/blob/main/package.json#L13-L14

Reply:
Let's stick with `./node_modules/.bin/performance-testing-dataset-extractor` for now